src/main/java/com/github/shyiko/mysql/binlog/event/deserialization/AbstractRowsEventDataDeserializer.java

```diff
@@ -478,6 +488,35 @@ private Serializable deserializeTimeInteger(int value) {
         return timestamp != null ? new java.sql.Timestamp(timestamp) : null;
     }
 
+    protected Serializable deserializeTimeV1New(ByteArrayInputStream inputStream) throws IOException {
```

Comment:
It would be great if we could document the format of value we return it in.
Yes, we can check MySQL source code for that but I think it would be better if we could keep the library self-contained.

I believe it is something like this:
```
3-byte (24-bit) integer of the following format.
The most significant bit represents a sign, the rest of the data is interpreted as decimal
DEC: HHMMSS
Examples: 
 - 0x020D07 -> 134407 -> 13:44:07 (need to validate)
 - TODO: add an example for negative time
```